### DISCUSSION TITLE
Cross-fade between 2 tracks

### DISCUSSION THREAD

**Discussion by JanJans640** [jellyfin/jellyfin-desktop#292] [Ideas]
Hi everybody,

First thanks for the great work on this fantastic project Jellyfin/Jellyfin-media-player.
Still one request from my side.
Is it possible to implement in jellyfin-media-player an option for cross-fade between 2 audio-tracks?
Would be great.
Thanks.

**Comment by iwalton3** [comment]
There’s no easy way to implement that in this player, but there are plenty of dedicated Jellyfin music players. As always PRs are welcome though

**Reply by JanJans640** [reply]
Thanks for your reply.
Best Regards.
